dask/multiprocessing.py

```diff
@@ -125,6 +125,15 @@ def pack_exception(e, dumps):
     return result
 
 
+def get_context():
+    """ Return the current multiprocessing context."""
+    if sys.platform == "win32" or sys.version_info.major == 2:
+        # Just do the default, since we can't change it:
```

Comment:
I feel like it should be an error if multiprocessing.context is set to a value that doesn't work on the platform

Reply:
The downside of that is you can't share a configuration file across platforms. If this is acceptable, I can make that change.

Alternatively... python warning/warning log message if the option is used on unsupported platform, instead of error?

Reply:
A warning can be an acceptable compromise.